src/Vulkan/VkDescriptorPool.cpp

```diff
@@ -101,8 +101,14 @@ BindlessHandle VkDescriptorPool::AllocateStaticDescriptor(const RHITexture& text
 
 BindlessHandle VkDescriptorPool::AllocateStaticDescriptor(const RHIBuffer& buffer)
 {
-    VEX_NOT_YET_IMPLEMENTED();
-    return BindlessHandle();
+    // Everything needs to be a storage buffer since we use ByteAddressBuffers
+    auto type = ::vk::DescriptorType::eStorageBuffer;
```

Comment:
Regarding this and what you mentioned in the PR comment about Vulkan forcing you to use ByteAddressBuffers, what is stopping us from using the buffer device address for all buffers instead of the bindless handle? Ok we increase the size of the handle to 64 bits, but we gain by avoiding a LOT of work on the CPU side, and making shader code-gen easier for ourselves. Also allows for having more than 2 billion bytes of buffers (which isn't that much in the grand scheme of data passed to the GPU), since we're no longer limited to using byteaddressbuffers with uint indexing.